constants.ts

```diff
@@ -0,0 +1,4 @@
+import { UnistylesRuntime } from "react-native-unistyles";
+
+export const searchBarContainerHeight =
+  (UnistylesRuntime.screen.height / 15) * 0.9;
```

Comment:
Dobra myśl, żeby mieć to zdefiniowane w jednym miejscu! Tylko mamy znowu problem z tym, że to zależy od runtime'u, który może się zmieniać, a wyliczane jest tylko raz. Za to używasz tego tylko wewnątrz createStyleSheet, gdzie zawsze dostajesz najbardziej aktualny runtime. Takze to mogłaby być funkcja, która przyjmuje runtime jako argument i zwraca wysokość jako wartość